src/glib/glib.c

```diff
@@ -0,0 +1,491 @@
+// Copyright 2015-2024 The NATS Authors
```

Comment:
With the exception of `glib/dispatch_pool.c` these were copied from `nats.c` with minimal changes